content/docs/about/contributing.md

```diff
@@ -94,8 +94,8 @@ should be fixed, you should own it. Here is how you get started.
 
 To find Kubeflow issues that make good entry points, look at the following tags:
 
-* [`good first issue`](https://github.com/kubeflow/kubeflow/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-* [`help wanted`](https://github.com/kubeflow/kubeflow/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+* [`good first issue`](https://github.com/kubeflow/kubeflow/issues?q=first+issue+label%3A%22good+first+issue%22)
```

Comment:
Thanks for fixing the links! This updated link isn't right, though. Based on @jlewi's comment on issue https://github.com/kubeflow/website/issues/1332, we need to put the links back to what they were before PR https://github.com/kubeflow/website/pull/1406.

Here's the link as it was before that PR:

```
[`good first issue`](https://github.com/kubeflow/kubeflow/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```